documentation/modules/creating-yaml-based-storage-maps-ui.adoc

```diff
@@ -0,0 +1,172 @@
+// Module included in the following assemblies:
+//
+// * documentation/doc-Migration_Toolkit_for_Virtualization/master.adoc
+
+:_content-type: PROCEDURE
+[id="creating-yaml-based-stoarge-maps-ui_{context}"]
+= Creating ownerless storage maps using YAML or JSON definitions in the {project-short} UI
+
+You can create ownerless storage maps by using YAML or JSON definitions in the {project-first} UI.
+
+.Prerequisites
+
+* Have a {vmw} source provider and a {virt} destination provider. For more information, see xref:adding-source-provider_vmware[Adding a VMware vSphere source provider] or xref:adding-source-provider_dest_vmware[Adding {a-virt} destination provider].
+
+.Procedure
+
+. In the {ocp} web console, click *Migration for Virtualization* > *Storage maps*.
+. Click *Create storage pap* > *Create with YAML*.
++
+The *Create StoragekMap* page opens.
```

Comment:
Typo here is well, random k added. Also should probably be "Create storage map" in the written out version, not in the kubernetes format

Reply:
@mmenestr Typo fixed. But for both YAML-based maps, the pages say "Create NetworkMap" and "Create StorageMap". I figured that was intentional, to follow the names of the CRs. 

![image](https://github.com/user-attachments/assets/2fe3fc2c-0f38-4114-9e30-ff7886691c81)

![image](https://github.com/user-attachments/assets/005c54e9-4f25-4963-a59b-d145ec2cbf77)